tools/proto_format/proto_format.sh

```diff
@@ -21,7 +21,7 @@ then
   bazel test ${BAZEL_BUILD_OPTIONS} //tools/protoxform:merge_active_shadow_test
 fi
 
-# Generate //versioning:active_protos.
+# Generate //versioning:active_protos and //versioning::frozen_protos.
```

Comment:
Yeah, you can put it as: `//versioning:frozen_protos` instead.

Reply:
ok.